release/release.go

```diff
@@ -1544,3 +1552,47 @@ func downloadArtifacts(v string, artifactNames []string) {
 		log.Fatalf("expect to download %d artifacts %s, only downloaded %d", len(artifactNames), artifactNames, numArtifactsDownloaded)
 	}
 }
+
+// We want the filenames we record in Papertrail to be unique, but the name of
+// the non-deb/RPM files are things like `release.zip` or `release.msi`. This
+// code will rename these files to something that matches the names that
+// _would_ be used if we were to upload them as official releases.
+//
+// This doesn't _guarantee_ uniqueness since we could run the exact same
+// commit in Evergreen twice, which would generate the same names, but it's
+// close enough. Papertrail will still record the information we give even if
+// we use the same filenames for two different CI runs.
+func renameReleaseFilesForPapetrail() {
+	pf, err := platform.GetFromEnv()
+	if err != nil {
+		log.Fatalf("could not find platform: %v", err)
+	}
+
+	v, err := version.GetCurrent()
+	if err != nil {
+		log.Fatalf("could not get version: %v", err)
+	}
+
+	glob := "release.*"
+	releaseFiles, err := filepath.Glob(glob)
+	if err != nil {
+		log.Fatalf("could not get files matching %q: %v", glob, err)
+	}
+
+	baseName := basenameForPlatformAndVersion(pf, v)
+	for _, file := range releaseFiles {
+		ext := filepath.Ext(file)
+		newName := baseName + ext
+		if err := os.Rename(file, newName); err != nil {
```

Comment:
I think the second param of `os.Rename` is a path instead of a filename. Is it intentional or an overlook?

Reply:
I'm not sure what you mean. This works as written. Here's the output from running this locally as a test:

```
renamed "release.tgz" to "mongodb-database-tools-ubuntu2204-x86_64-100.9.4-34-g427bcf56-dirty.tgz"
```

That's exactly how we want the rename to work.

Reply:
A "just a file name" is still a path. It's _not_ a fully-qualified path. This all works fine because this code is run in the same directory as the files being renamed, so we don't need to provide a longer path.

Reply:
Thanks, that's why my question was if this is intentional or an overlook?